app/src/game.ts

```diff
@@ -329,7 +335,9 @@ export class Game {
   private _addDungeonToScene(): void {
     this._dungeon = new Dungeon();
     const textureLoader = new THREE.TextureLoader();
-    const wallTexture = textureLoader.load('./img/wall.jpg');
+    const wallTexture = textureLoader.load(
```

Comment:
Warum hast du das ausgetauscht? Gefällt dir deine Wand nicht mehr? ^^
Und bitte dann die datei in den public folder kopieren und von dort laden.

Reply:
bei der alten wand hat man nicht wirklich gewusst wo man ist und das neue img macht es vielleicht besser gehört aber besprochen und dann festgelegt